cmake/Modules/LocalLibyuv.cmake

```diff
@@ -1,4 +1,4 @@
-set(AVIF_LIBYUV_TAG "a6a2ec654b1be1166b376476a7555c89eca0c275")
+set(AVIF_LIBYUV_TAG "d32d19ccf2711d8bdf43a68dc5f8daedb7965c62")
```

Comment:
This commit has not been rolled into Chrome. My roll attempt was reverted because it broke some tests that compare the conversion results with golden images: https://chromium-review.googlesource.com/c/chromium/src/+/6230822

So we could consider updating to the parent of this commit first.

Note: This commit was rolled into Google's internal repository last night and it has not been reverted.

Reply:
> This commit has not been rolled into Chrome. My roll attempt was reverted because it broke some tests that compare the conversion results with golden images: https://chromium-review.googlesource.com/c/chromium/src/+/6230822
> 
> So we could consider updating to the parent of this commit first.
> 
> Note: This commit was rolled into Google's internal repository last night and it has not been reverted.

Thanks for pointing that out. I rolled back to the parent.